website/content/en/docs/deployment/prerequisites.md

```diff
@@ -81,5 +81,28 @@ The command above installs the following tools:
 - [terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli) - An infrastructure as code tool that lets you develop cloud and on-prem resources.
 - [helm](https://helm.sh/docs/intro/install/) - A package manager for Kubernetes
 
+## Configure AWS CLI
+
+To access AWS services with the AWS CLI, you need an AWS account, IAM credentials, and an IAM access key pair. When running AWS CLI commands, the AWS CLI needs to have access to those AWS credentials. 
```

Comment:
To access AWS services, you need an AWS account and setup IAM credentials. Follow https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html to setup your IAM credentials.

Note that your IAM user/role must have Administrative Privileges. Once your configuration is complete, run `aws sts get-caller-identity` to verify AWS CLI has access to AWS credentials